test/forking/infura/simple.js

```diff
@@ -0,0 +1,156 @@
+const shouldSkip = require("./skip");
+const Web3 = require("web3");
+var assert = require("assert");
+const Ganache = require(process.env.TEST_BUILD
+  ? "../../../build/ganache.core." + process.env.TEST_BUILD + ".js"
+  : "../../../index.js");
+
+const logger = {
+  log: function(msg) {
+    /* console.log(msg) */
+  }
+};
+
+describe("Simple Infura", function() {
+  let INFURA_KEY;
+
+  before(function() {
+    if (shouldSkip()) {
+      this.skip();
+    } else {
+      INFURA_KEY = process.env.INFURA_KEY;
+    }
+  });
+
+  it("forks mainnet https", async function() {
+    // https://etherscan.io/block/10661638
+    const blockHeight = 10661638;
```

Comment:
Does the infura key have archival access? Will this blockHeight need to be adjusted to stay viable?

Reply:
It does have archival access; there's no need to keep it updated